modules/Presentation/README.md

```diff
@@ -0,0 +1,10 @@
+The only important file is presentation.py
+
+Helpers.py contains a few helper functions.
+
+test_data_generator.py simply returns test_data.
+
+Purgatory.py is for old / no longer relevant code / earlier versions of stuff.
+Its just to show the progress, and that I've been through a few iterations before I landed on whats in presentation.py
+Just so it doesnt look like I've spent a week on a few filter methods and a few simple .plot functions.
+For some reason I had a hard time getting the structure right.
```

Comment:
The readme should not be here either :) This is a comment for either the pull request or the issue on github :) and we know you haven't spent a week on a few filter methods :)